signupsignin.js

```diff
@@ -0,0 +1,130 @@
+
+const firebaseConfig = {
```

Comment:
you need to remove this config from here and use it as i wrote above.

Reply:
done